sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/Data.java

```diff
@@ -625,7 +639,7 @@ public void onCallFailed(Exception e) {
      * Create a document
      * The document type (T) must be JSON deserializable
      */
-    private synchronized <T> AppCenterFuture<PaginatedDocuments<T>> instanceList(final Class<T> documentType, final String partition) {
+    private synchronized <T> AppCenterFuture<PaginatedDocuments<T>> instanceList(final Class<T> documentType, final String partition, final ReadOptions readOptions) {
```

Comment:
where is it used?

Reply:
It will be used in a later pr when we query local storage.  We are going to use the ReadOptions methods to clean up our current implementation.

Reply:
PRs have to be self consistent, this is an unused variable right now.
You can make a PR on top of another PR, but then we cannot merge this one.